infra/gcp/ensure-main-project.sh

```diff
@@ -57,6 +57,11 @@ CLUSTER_USERS_GROUP="gke-security-groups@kubernetes.io"
 # The DNS admins group.
 DNS_GROUP="k8s-infra-dns-admins@kubernetes.io"
 
+# Buckets for the logs of prow
+PROW_BUCKETS=(
+    k8s-prow-staging-logs
+)
```

Comment:
If the intent is to use this like gs://kubernetes-jenkins, I feel like we're going to want those sorts of buckets not in kubernetes-public.  It'll be easier to keep track of CI artifact costs if they're in their own project.  I'm open to suggestions.

Reply:
The PR is more in favor of the support multiple buckets by tide. I didn't consider the billing aspect.